src/vl-upload.js

```diff
@@ -247,6 +247,10 @@ export class VlUpload extends vlFormValidationElement(vlElement(HTMLElement)) {
       this._dressFormValidation();
       this._dropzone.on('addedfile', () => setTimeout(() => this.dispatchEvent(new Event('change'))));
       this._dropzone.on('removedfile', () => setTimeout(() => this.dispatchEvent(new Event('change'))));
+      this._dropzone.on('success', (file, response) => {
+      	file.responseData = response;
+      	setTimeout(() => this.dispatchEvent(new Event('change')));
```

Comment:
Is die setTimeout echt nodig hier?

Reply:
consistent met de rest gehouden ...

Reply:
Ja ik zie het. 
Misschien dan niet per se iets voor deze PR, maar eventueel in de toekomst eens herbekijken waarom die setTimeouts hier in staan, @LucasSledsens en @meirege .

Reply:
er wordt geen delay meegegeven. Dan vallen we dus terug op de volgende 'tick'

"... If this parameter is omitted, a value of 0 is used, meaning execute "immediately", or more accurately, the next event cycle. Note that in either case, the actual delay may be longer than intended; see ..."

Reply:
Ja, dat weet ik, maar het is me onduidelijk waarom we dit willen bij het dispatchen van een event, op deze plaats.

Reply:
ja zie er ook niet echt het nut van in

Reply:
ik heb een triggerChange methode geextract, dat kan dan idd later eens verder bekijken worden waarom daar een setTimeout bij staat